Bookish/Views/Catalog/CopyList.cshtml

```diff
@@ -0,0 +1,63 @@
+﻿@using Microsoft.EntityFrameworkCore.Metadata.Internal
+@model Bookish.Models.CopyViewModel
+
+@{
+    ViewBag.Title = "Copy List";
+}
+
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.3/font/bootstrap-icons.css">
+        <link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.12.1/css/jquery.dataTables.css">
+        <script type="text/javascript" charset="utf8" src="https://cdn.datatables.net/1.12.1/js/jquery.dataTables.js"></script>
+        <title>BookCopies</title>
+    </head>
+    <body>
+    <div>
+        @{ 
+            Book? foundBook; 
+            Author? foundAuthor; 
+        }
+        @using (var context = new LibraryContext())
+        {
+            foundBook = context.Books.SingleOrDefault(a =>
+                a.BookID == Model.CatalogEntries.First().BookID);
+            foundAuthor = context.Authors.SingleOrDefault(a => a.AuthorID == foundBook.AuthorID);
+        }
```

Comment:
Ooooh this logic should definitely not go in the view. If you want to show information about a book and author, you should provide this from the backend via the view model - you'll want to change your `CopyViewModel` to contain this info, and add some logic wherever you create this model to provide the info

Reply:
Oooops yeah this was very silly of me - in hindsight I have no idea why I did it this way since its different to every other one of the ways we got the data from the server :)